central/networkbaseline/manager/manager_impl.go

```diff
@@ -857,6 +860,8 @@ func (m *manager) addBaseline(deploymentID, deploymentName, clusterID, namespace
 		return err
 	}
 
+	flows = m.enrichFlows(clusterID, deploymentID, deploymentName, namespace, flows)
```

Comment:
I see that the first 4 arguments are only used to create a `map[string]*storage.ListDeployment` with a single element, maybe it makes sense to create the `storage.ListDeployment` here and pass it to `m.enrichFlows`?
```suggestion
	deployment := &storage.ListDeployment{
		deploymentID: &storage.ListDeployment{
			Name:      deploymentName,
			Id:        deploymentID,
			ClusterId: clusterID,
			Namespace: namespace,
		},
	}

	flows = m.enrichFlows(deployment, flows)
```

Then in `enrichFlows` you can just do something like:
```go
listDeploymentMap := map[string]*storage.ListDeployment{
	deployment.Id: deployment,
}

Reply:
Fair point; done!